### DISCUSSION TITLE
Alternative List View for Tiles

### DISCUSSION THREAD

**Discussion by fronbow** [Laravel-Backpack/community-forum#1014] [Q&A (Help)]
Hi All

I'm trying to work out the logic for a custom operation that is basically an alternative list view. 
I initially thought it was just a custom operation but the more I think about it, it needs all the functionality of the ListOperation but decoupled from dataTables.
[Isotope](https://isotope.metafizzy.co/) seems to be a good fit in that it will allow me to have tile-able records.
[Tabulator](https://tabulator.info/) might also be a nice alternative option to dataTables.

Each tile will be a record with buttons for modal operations (the use-case is to record when people enter/leave our day centre so that staff can record any interactions the clients have in the day and then write overall notes up at end of service).

Can I just duplicate the `src/app/Http/Controllers/Operations/ListOperation.php` file and its' view and rename it to `TileOperation` (obviously editing the source with relevant changes) and then add that Operation to my relevant `CrudController`? Or am I better off implementing a new Operation?

Cheers for any help/ideas

**Comment by jcastroa87** [comment]
Hello @fronbow,

If you need to replace the list view, copy the file located at /vendor/backpack/crud/src/resources/views/crud/list.blade.php to /resources/views/vendor/backpack/crud/. After this, you can start customizing the view as needed. Please note that most of the DataTables functions depend on the script at /vendor/backpack/crud/src/resources/views/crud/inc/datatables_logic.blade.php.

Alternatively, you can override the list controller, but be aware that this change will affect all CRUD operations in the application.

```
<?php

namespace App\Providers;

use Illuminate\Support\ServiceProvider;

class AppServiceProvider extends ServiceProvider
{
    /**
     * Register any application services.
     *
     * @return void
     */
    public function register()
    {
        // override register controller
        $this->app->bind(
            \Backpack\CRUD\app\Http\Controllers\Operations\ListOperation::class,
            \App\Http\Controllers\Operations\ListOperation::class
        );
    }
    
    //
}

```

Cheers.

**Reply by fronbow** [reply]
Cheers @jcastroa87, for this particular problem I'm replacing `use \Backpack\CRUD\app\Http\Controllers\Operations\ListOperation;` at the top of the CrudController with `use \App\Http\Controllers\Admin\Operations\TileOperation;` and just working through changing a copy of the bp ListOperation into a TileOperation.
I can see at the bottom of the list.blade.php file it's loading some dataTables css files and the datatables_logic blade file so I figured if I take them out and substitute with whichever other library I need then in theory it should be closer to working!

It would be nice if bp could be decoupled from dataTables as there are some good alternatives out there from my searches recently (I'm liking the aesthetics of [Tabulator](https://tabulator.info/)!)

**Comment by jcastroa87** [comment]
I don't think that will happen anytime soon, most of our users use DataTables for their lists and we have developed many features for it. We are looking to have Livewire components in the future and that is our priority.

If something works, don't touch it ;)

I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
